### PR TITLE
US975260: Refactor HPE references

### PR DESCRIPTION
**Pull request by raghavendran-gopalakrishnan**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=975260

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -90,67 +90,67 @@
...
                 <artifactId>worker-queue-rabbit</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
             </dependency>
             <dependency>
-                <groupId>com.github.workerframework</groupId>
+                <groupId>com.github.workerframework.worker.datastores</groupId>
                 <artifactId>worker-store-http</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:150]:
```suggestion
                <version>9.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 2]
```diff
@@ -90,67 +90,67 @@
...
                 <artifactId>worker-configs</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
             </dependency>
             <dependency>
-                <groupId>com.github.workerframework</groupId>
+                <groupId>com.github.workerframework.worker.queues</groupId>
                 <artifactId>worker-queue-rabbit</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:145]:
```suggestion
                <version>9.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 3]
```diff
@@ -90,67 +90,67 @@
...
                 <artifactId>worker-api</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
             </dependency>
             <dependency>
-                <groupId>com.github.workerframework</groupId>
+                <groupId>com.github.workerframework.worker</groupId>
                 <artifactId>worker-configs</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:140]:
```suggestion
                <version>9.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 4]
```diff
@@ -90,67 +90,67 @@
...
                 <artifactId>util-rabbitmq</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
             </dependency>
             <dependency>
-                <groupId>com.github.workerframework</groupId>
+                <groupId>com.github.workerframework.worker</groupId>
                 <artifactId>worker-api</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:135]:
```suggestion
                <version>9.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 5]
```diff
@@ -90,67 +90,67 @@
...
                 <artifactId>util-ref</artifactId>
-                <version>3.0.0-549</version>
+                <version>4.0.0-SNAPSHOT</version>
             </dependency>
             <dependency>
-                <groupId>com.github.workerframework</groupId>
+                <groupId>com.github.workerframework.worker.util</groupId>
                 <artifactId>util-rabbitmq</artifactId>
-                <version>8.0.0-1196</version>
+                <version>9.0.0-975260-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:130]:
```suggestion
                <version>9.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 6]
```diff
@@ -21,9 +21,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.workerframework.testing</groupId>
+    <groupId>com.github.workerframework.worker.testing</groupId>
```
Comment (dermot-hardy) [inline pom.xml:24]: Do we need to be adding the `.worker` segment for this repository?

Comment (raghavendran-gopalakrishnan) [reply pom.xml:24]: Thought to keep it aligned with https://github.com/WorkerFramework/worker-framework/pull/204/files. But lets discuss if its required.

File: pom.xml [hunk 7]
```diff
@@ -90,69 +90,69 @@
                 <version>6.5.1</version>
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi</groupId>
+                <groupId>com.github.cafapi.common.api</groupId>
```
Comment (dermot-hardy) [inline pom.xml:93]: It seems we made a mistake in `caf-common` on this.

File: pom.xml [hunk 8]
```diff
@@ -90,69 +90,69 @@
                 <version>6.5.1</version>
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi</groupId>
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>caf-api</artifactId>
-                <version>3.0.0-549</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:95]:
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 9]
```diff
@@ -90,69 +90,69 @@
...
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi.cipher</groupId>
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>cipher-null</artifactId>
-                <version>3.0.0-549</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:100]:
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 10]
```diff
@@ -90,69 +90,69 @@
...
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi.config</groupId>
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>config-decoder</artifactId>
-                <version>3.0.0-549</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:105]:
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 11]
```diff
@@ -90,69 +90,69 @@
...
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi.config</groupId>
-                <artifactId>config-system</artifactId>
-                <version>3.0.0-549</version>
+                <groupId>com.github.cafapi.common</groupId>
+                <artifactId>bootstrapconfig-system</artifactId>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:110]:
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 12]
```diff
@@ -90,69 +90,69 @@
...
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi.util</groupId>
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>util-moduleloader</artifactId>
-                <version>3.0.0-549</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:115]:
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 13]
```diff
@@ -90,69 +90,69 @@
...
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi.util</groupId>
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>util-naming</artifactId>
-                <version>3.0.0-549</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:120]:
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

File: pom.xml [hunk 14]
```diff
@@ -90,69 +90,69 @@
...
             </dependency>
             <dependency>
-                <groupId>com.github.cafapi.util</groupId>
+                <groupId>com.github.cafapi.common</groupId>
                 <artifactId>util-ref</artifactId>
-                <version>3.0.0-549</version>
+                <version>4.0.0-975260-1-SNAPSHOT</version>
```
Comment (raghavendran-gopalakrishnan) [inline pom.xml:125]:
```suggestion
                <version>4.0.0-SNAPSHOT</version>
```

**Review by dermot-hardy — APPROVED** [review]

**Pull request merged**